tests/unit/s2n_cert_validation_callback_test.c

```diff
@@ -187,16 +187,6 @@ int main(int argc, char *argv[])
                 .data = { .call_accept_or_reject = true, .accept = false, .return_success = false },
                 .expected_error = S2N_ERR_CANCELLED
             },
-            {
-                .data = { .call_accept_or_reject = false, .return_success = false },
```

Comment:
Is `call_accept_or_reject` needed now that it's always set to true? We can probably just have the sync/async tests each decide if they should call accept/reject.